docs/metrics/metrics-operators/predict.md

```diff
@@ -0,0 +1,67 @@
+---
+id: predict
+title: predict Metrics Operator
+sidebar_label: predict
+description: Predict future values based on a single time series.
+keywords:
+    - metrics
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The `predict` operator takes a single time series metric to predict future values. Predicting metrics such as CPU Usage or memory consumption can be useful for resource and capacity planning use cases.
+
+`predict` supports linear regression (linear) models, which use a linear model on the timestamp to extrapolate into the future, and Auto-regressive (ar) models, which use a window of previously observed data to predict future values. Note that prediction using an AR model does not output any predictions in the first time window.
+
+The `predict` operator outputs two time series: the original input time series and the predicted time series that extends into the future. The predicted time series is also depicted over a portion of the historical time range so that the user can validate forecast accuracy at a glance against actual values.   
+
+## Syntax
+```sql
+… | predict [model=<model>] [forecast=<forecast>] [ar.window=<ar.window>]
+```
+
+Where:
+
+* `model` specifies the type of regression you want to perform:
```

Comment:
What's the default in case no model param is specified ?

Reply:
updated to state default is linear.